### ISSUE TITLE
Add tip about aliases to Core Concepts - Assets

### ISSUE THREAD

**Issue by carlcs** [craftcms/docs#154]
https://docs.craftcms.com/v3/assets.html

**Comment by brandonkelly** [comment]
Good call, just rewrote portions of that page to point out that aliases can be used.

**Comment by carlcs** [comment]
You write wonderful documentation Brandon! Craft 3 docs have improved a lot since its launch. Thanks

**Comment by brandonkelly** [comment]
Thank you!
